tests/models/test_google.py

```diff
@@ -1828,3 +1834,184 @@ class CityLocation(BaseModel):
     agent = Agent(m, output_type=PromptedOutput(CityLocation), builtin_tools=[UrlContextTool()])
     result = await agent.run('What is the largest city in Mexico?')
     assert result.output == snapshot(CityLocation(city='Mexico City', country='Mexico'))
+
+
+@pytest.mark.parametrize(
```

Comment:
It would be great if we had a minimal reproducible example that triggers the error against the actual API, so we can verify what parts of the fix are necessary (e.g. is sending back thinking parts enough? does a one-space string work? is it only Vertex or also GLA?). Right now, we're effectively duplicating the fix in tests, rather than verifying that a previously-failing example now succeeds.